Formula/wsk.rb

```diff
@@ -0,0 +1,21 @@
+class Wsk < Formula
+  desc "OpenWhisk CLI"
+  homepage "https://console.ng.bluemix.net/openwhisk/cli"
+  url "https://openwhisk.ng.bluemix.net/cli/go/download/mac/amd64/OpenWhisk_CLI-mac.zip"
+  version "0.1.0"
+
+  def install
+    bin.install "wsk"
+  end
+
+  def caveats
+    <<-EOS.undent
+      Until OpenWhisk project does not provide an official version for the CLI tools,
+      to upgrade we must uninstall/install the formula.
+    EOS
+  end
+
+  test do
+    system "true"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!